### PR TITLE
Debezium Postgres demo.

### PR DESCRIPTION
**Pull request by ryzhyk**
Added a demo reading from a Debezium Postgres source connector.  It is almost identical to the Debezium MySQL demo.

Is this a user-visible change (yes/no): no

### CODE REVIEW

File: demo/project_demo13-DebeziumPostgres/run.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+         ]
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--api-url", required=True, help="Feldera API URL (e.g., http://localhost:8080 )")
+    parser.add_argument("--kafka-url-from-pipeline", default="redpanda:9092", help="Kafka broker address reachable from the pipeline")
+    parser.add_argument("--kafka-url-from-script", default="localhost:19092", help="Kafka broker address reachable from this script")
+    parser.add_argument('--start', action='store_true', default=False, help="Start the Feldera pipeline")
+    args = parser.parse_args()
+    create_debezium_mysql_connector(args.kafka_url_from_script)
```
Comment (snkas) [inline demo/project_demo13-DebeziumPostgres/run.py:26]: create_debezium_postgres_connector

**Review by snkas — APPROVED** [review]

File: demo/project_demo13-DebeziumPostgres/run.py (hunk 1 above)
Comment (ryzhyk) [reply demo/project_demo13-DebeziumPostgres/run.py:26]: good catch, thanks!

File: demo/project_demo13-DebeziumPostgres/project.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
+-- SQL program from the Debezium MySQL tutorial used to demo Feldera
+-- Debezium integration.
+
+CREATE TABLE customers (
+  id int NOT NULL PRIMARY KEY,
+  first_name varchar(255) NOT NULL,
+  last_name varchar(255) NOT NULL,
+  email varchar(255) NOT NULL
+) with ('materialized' = 'true');
```
Comment (mihaibudiu) [inline demo/project_demo13-DebeziumPostgres/project.sql:9]: why do we need to materialize these tables?

**Review by mihaibudiu — APPROVED** [review]

File: demo/project_demo13-DebeziumPostgres/project.sql (hunk 2 above)
Comment (ryzhyk) [reply demo/project_demo13-DebeziumPostgres/project.sql:9]: Just for debugging

**Pull request merged**